config/authorities/licenses.yml

```diff
@@ -41,9 +41,12 @@ terms:
   - id: http://creativecommons.org/publicdomain/zero/1.0/
     term: CC0 1.0 Universal
     active: true
-  - id: http://www.europeana.eu/portal/rights/rr-r.html
+  - id: https://rightsstatements.org/page/InC/1.0/?language=en
```

Comment:
@srerickson I'd remove the `/?language=en` at the end of the url.

Reply:
@awead, it's actually necessary: if you remove the URL param, the link won't resolve.

Reply:
actually this works: https://rightsstatements.org/page/InC/1.0/
but not this: https://rightsstatements.org/page/InC/1.0

bad.